eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/CdcDataPublisher.java

```diff
@@ -66,7 +80,7 @@ public void stop() {
       producer.close();
   }
 
-  public void handleEvent(EVENT publishedEvent) throws EventuateLocalPublishingException {
+  public Optional<CompletableFuture<?>> handleEvent(EVENT publishedEvent) throws EventuateLocalPublishingException {
```

Comment:
The logic would be simpler if this always returned a CompletableFuture<> instead of an Optional

Reply:
Ok, I will remove optional